src/invoke_training/lora/injection/stable_diffusion_v1.py

```diff
@@ -25,6 +25,7 @@ def inject_lora_into_unet_sd1(unet: UNet2DConditionModel) -> LoRALayerCollection
         include_descendants_of={Transformer2DModel},
         exclude_descendants_of=None,
         prefix="lora_unet",
+        dtype=torch.float32,
```

Comment:
I may re-visit this in the future, but for now mixed-precision training requires that the trainable weights are initially loaded in full precision.